docs/react-testing-library/api.md

```diff
@@ -167,7 +167,7 @@ This is a simple wrapper around `prettyDOM` which is also exposed and comes from
 
 It'd probably be better if you test the component that's doing the prop updating
 to ensure that the props are being updated correctly (see
-[the Guiding Principles section](#guiding-principles)). That said, if you'd
+[the Guiding Principles section](/guiding-principles)). That said, if you'd
```

Comment:
Hmm this isn't right - see the deploy preview. I think it should be `/docs/guiding-principles.md`

Reply:
Oops, Correct it!